quickstarts/blazor-webassembly/config.yml

```diff
@@ -0,0 +1,53 @@
+id: d47d1298-fe2a-437b-91f1-c0316bd274d7
+
+
+slug: blazor-webassembly
+description: |
+  ## Comprehensive monitoring quickstart for Blazor WebAssembly
+
+  As you probably know, Microsoft developed Blazor WebAssembly to build web UI via C#, HTML, and CSS. With our Browser Monitoring agent, you can monitor your Blazor WebAssembly app performance with our quickstart dashboards right out of the box.
+  
+
+  ## Why monitor Blazor WebAssembly?
+  
+  There are a few strong arguments in favour of using Blazor WebAssembly:
+
+  - Monitoring the Higher Letency and Slower Application.
+  - Using New Relic Browser Monitoring agent capture the Page load time, Web transactions and Https response.
```

Comment:
```suggestion
  - Using New Relic Browser Monitoring agent captures the Page load time, Web transactions and Https response.
```

Reply:
Hi @brammerl ,
Updated text changes, Kindly review it.

Thank you...